node/package.json

```diff
@@ -36,7 +36,7 @@
     "deeper": "",
     "fs-promise": "",
     "group-by": "",
-    "nodegit": "^0.21.1",
+    "nodegit": "^0.23.0",
```

Comment:
Why ^ ? Are the floating versions the reason you added the lockfile?

Reply:
The `^` defines which version we support. The lock file basically means the what package is currently used. Ideally all patches should be supported, but lock file makes version bumps explicit, so if error happens, it is easy to track or git-bisect to find out the issue.